src/Reader/Extension/AbstractFeed.php

```diff
@@ -115,12 +115,12 @@ public function toArray() // untested
         return $this->data;
     }
 
-    /**
-     * Set the XPath query
-     *
-     * @param  DOMXPath $xpath
-     * @return AbstractEntry
-     */
+	/**
```

Comment:
Please revert the changes for indentation.

Reply:
Weird. I don't know how that happened, my editor should have been set to spaces.  I'll clean these up.